scripts/v2-migration/products/upload-product-ipfs-metdata.js

```diff
@@ -0,0 +1,137 @@
+require('dotenv').config();
+const ipfsClient = require('ipfs-http-client');
+const fs = require('fs');
+const path = require('path');
+const fetch = require('node-fetch');
+const { parse: csvParse } = require('csv-parse/sync');
+
+async function readFileFromURL(url) {
+  const file = await fetch(url).then(x => x.buffer());
+  return file;
+}
+
+const CoverWordings = {
+  protocol: 'https://uploads-ssl.webflow.com/62d8193ce9880895261daf4a/63d0f4c4cca088730ac54ccc_ProtocolCoverv1.0.pdf',
+  custodian:
+    'https://uploads-ssl.webflow.com/62d8193ce9880895261daf4a/63d0f4d7b378db634f0f9a9d_CustodyCoverWordingv1.0.pdf',
+  token: 'https://uploads-ssl.webflow.com/62d8193ce9880895261daf4a/63d0f475a1a2c7250a1e9697_YieldTokenCoverv1.0.pdf',
+  sherlock:
+    'https://uploads-ssl.webflow.com/62d8193ce9880895261daf4a/63d0f7c4f0864e48c46ad93c_SherlockExcessCoverv1.0.pdf',
+  eth2slashing:
+    'https://uploads-ssl.webflow.com/62d8193ce9880895261daf4a/63d0f8390352b0dc1cb8112b_ETH2-Staking-Cover-Wording-v1.0.pdf',
+  liquidcollective:
+    'https://uploads-ssl.webflow.com/62d8193ce9880895261daf4a/63d7cb35dea9958c3952e9c0_Liquid-Collective-v1.0.pdf',
+};
+
+const IPFS = {
+  API: {
+    url: 'https://api.nexusmutual.io/ipfs-api/api/v0',
+  },
+  GATEWAY: 'https://api.nexusmutual.io/ipfs/',
+};
+
+async function uploadCoverWordingForProductType(ipfs, productType) {
+  const url = CoverWordings[productType];
+  console.log(`Fetching ${productType} cover wording from ${url}..`);
+  const agreementBuffer = await readFileFromURL(url);
+
+  console.log(`Uploading ${productType} cover wording to IPFS..`);
+  const agreement = await ipfs.add(agreementBuffer);
+  const productTypeHash = agreement.path;
+
+  console.log(`Pinning ${productTypeHash}`);
+  await ipfs.pin.add(productTypeHash);
+
+  console.log(`Succesfully pinned`);
+  return productTypeHash;
+}
+
+/**
+   Expected format:
+
+ Exclusions that apply but are not limited to:
+ - Losses due to a compromised wallet;
+ - Losses due to a previously disclosed vulnerability;
+ */
```

Comment:
`Exclusions that apply but are not limited to:` is unnecessary.
Each bullet point should be an item in the array

Reply:
Yes that is how it's stored but the original CSV followed that format.

So I am always stripping out that prefix and throwing it away.

Reply:
eliminated now from original file each exclusion is on its own line